test/built-ins/Array/from/mapfn-returns-integer.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: mapFn returns a non-integer
+es6id: 22.1.2.1
+---*/
+
+var array = [ 2, 4, 8, 16, 32, 64, 128 ];
+var mapObj = {

```

Comment:
The use of `mapObject` here does not seem necessary. Instead, you could just specify a `mapFn` declaration and a variable for `arrayIndex`. This would make the test more direct, easier to read, and easier to debug.